spark/src/main/scala/io/delta/tables/execution/DeltaTableOperations.scala

```diff
@@ -112,6 +113,41 @@ trait DeltaTableOperations extends AnalysisHelper { self: DeltaTable =>
     toDataset(sparkSession, restore)
   }
 
+  protected def executeClone(
+    table: DeltaTableV2,
+    target: String,
+    replace: Boolean,
+    tableProperties: Map[String, String],
+    versionAsOf: Option[Long] = None,
+    timestampAsOf: Option[String] = None
```

Comment:
```suggestion
    timestampAsOf: Option[String] = None): DataFrame = withActiveSession(sparkSession) {
```

Like the style guide as well

Reply:
Updated